src/client/util.js

```diff
@@ -16,19 +17,21 @@ export function isAndroidApp () {
   return typeof AndroidBridge !== 'undefined'
 }
 
-/* usage:
- * var connectWebViewJavascriptBridge = require('webViewJavascriptBridge')
- * connectWebViewJavascriptBridge(function(bridge) {
- *   bridge.send(...)
- * }
- */
-export function connectWebViewBridge (callback) {
-  if (window.WebViewJavascriptBridge) {
-    callback(window.WebViewJavascriptBridge)
-  } else {
-    document.addEventListener('WebViewJavascriptBridgeReady', function () {
-      callback(window.WebViewJavascriptBridge)
-    }, false)
+export function calliOSBridge (message, callback) {
+  if (isiOSApp()) {
```

Comment:
can you just short-circuit the method here with `if (!isIOSApp()) return` to reduce indentation?